src/sentry/workflow_engine/migration_helpers/issue_alert_conditions.py

```diff
@@ -71,10 +74,15 @@ def create_existing_high_priority_issue_data_condition(
 
 
 @data_condition_translator_registry.register(EventAttributeCondition.id)
+@data_condition_translator_registry.register(EventAttributeFilter.id)
 def create_event_attribute_data_condition(
     data: dict[str, Any], dcg: DataConditionGroup
 ) -> DataCondition:
-    comparison = {"match": data["match"], "value": data["value"], "attribute": data["attribute"]}
+    comparison = {
+        "match": data.get("match"),
+        "value": data.get("value"),
+        "attribute": data.get("attribute"),
+    }
```

Comment:
probably safer to do it this way in case comparison is malformed?

Reply:
hmm if they don't exist we should loudly error somehow

Reply:
we don't have to do anything more fancy right now, but if you get the key directly it will error if it doesn't exist